bricks/template/__brick__/{{project_name.snakeCase()}}/android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
```

Comment:
Please avoid adding unrelated changes to this task. I think you should base this PR on my one https://github.com/nimblehq/flutter-templates/pull/201 to extend the updates.